lib.ftd

```diff
@@ -2683,3 +2683,337 @@ link if { sneak-peak.cta-url != NULL }: $sneak-peak.cta-url
+body body: 
+optional ftd.image-src src:
+
+-- ftd.column:
+width: fill-container
+align-self: center
+border-width.px: 1
+border-color: $inherited.colors.custom.one
+border-right-width.px: 10
+border-bottom-width.px: 10
+margin-top.px: 63
+padding-horizontal.px: 30
+padding-top.px: 30
+background.solid: $inherited.colors.background.step-1
+padding-bottom.px: 373
+overflow-x: hidden
+
+-- ftd.column:
+width: fill-container
+
+-- ftd.text: $about-us-mobile.title
+role: $inherited.types.heading-medium
+color: $inherited.colors.text
+region: h3
+margin-bottom.px: 8
+
+-- ftd.text: 
+role: $inherited.types.copy-regular
+color: $inherited.colors.text
+classes: markdown
+
+$about-us-mobile.body
+
+-- end: ftd.column
+
+-- ftd.column:
+if: { about-us-mobile.src != NULL }
+anchor: parent
+bottom.px: 0
+right.px: -42
+
+-- ftd.image:
+src: $about-us-mobile.src
+width.fixed.px: 373
+height.fixed.px: 340
+
+-- end: ftd.column
+
+-- end: ftd.column
+
+-- end: about-us-mobile
+
+
+
+
+
+
+
+
+
+
+-- component our-team:
+caption title:
+$commons.team list members-list:
+
+-- ftd.column:
+width: fill-container
+align-content: center
+margin-top.px: 86
+
+-- ftd.column:
+width: fill-container
+max-width.fixed.px: 1320
+
+-- ftd.column:
+width: fill-container
+min-height.fixed.px: 244
+
+-- ftd.image: 
+if: { ftd.device != "mobile" }
+src: $assets.files.static.about-us.our-team.svg
+anchor: parent
+right.px: 22
+top.px: -16
+
+-- ftd.image: 
+if: { ftd.device == "mobile" }
+src: $assets.files.static.about-us.our-team.svg
+anchor: parent
+right.px: 0
+top.px: 0
+
+-- ftd.text: $our-team.title
+if: { ftd.device != "mobile" }
+role: $inherited.types.heading-medium
+color: $inherited.colors.text
+margin-top.px: 188
+margin-left.px: 2
+
+-- ftd.text: $our-team.title
+if: { ftd.device == "mobile" }
+role: $inherited.types.heading-medium
+color: $inherited.colors.text
+margin-top.px: 260
+margin-left.px: 12
+
+-- end: ftd.column
+
+-- ftd.row:
+if: { ftd.device != "mobile" }
+padding-bottom.px: 150
+margin-top.rem: 3
+margin-left.px: -10
+
+-- member-card: $obj.title
+$loop$: $our-team.members-list as $obj
+designation: $obj.designation
+avatar: $obj.avatar
+link: $obj.link
+
+-- end: ftd.row
+
+-- ftd.column:
+if: { ftd.device == "mobile" }
+padding-bottom.px: 150
+margin-top.rem: 3
+margin-left.px: -10
+
+-- member-card: $obj.title
+$loop$: $our-team.members-list as $obj
+designation: $obj.designation
+avatar: $obj.avatar
+link: $obj.link
+
+-- end: ftd.column
+
+-- end: ftd.column
+
+-- end: ftd.column
+
+-- end: our-team
+
+
+
+
+
+
+
+
+
+
+-- component member-card:
+caption title:
+string designation:
+ftd.image-src avatar:
+string link:
+
+-- ftd.column:
+width: fill-container
+min-width.fixed.percent if {ftd.device != "mobile"}: 33.33
+padding-horizontal.px if {ftd.device != "mobile"}: 12
+padding-horizontal.px if {ftd.device == "mobile"}: 20
+
+-- ftd.column:
+width: fill-container
+padding.px: 20 
+background.solid: $inherited.colors.background.step-1
```

Comment:
```suggestion
-- ftd.column:
width: fill-container
min-width.fixed.percent if {ftd.device != "mobile"}: 33.33
padding-horizontal.px if {ftd.device != "mobile"}: 32
padding-horizontal.px if {ftd.device == "mobile"}: 40
padding-vertical.px: 20
background.solid: $inherited.colors.background.step-1
```

Reply:
@Arpita-Jaiswal two columns are required here as I tried to merge them the UI is breaking.
Basically UI is like this: https://admint.io/#/about